modules/ROOT/pages/copy-and-paste.adoc

```diff
@@ -0,0 +1,38 @@
+= Copy & paste options
+
+:title_nav: Copy and paste options
+:description: TinyMCE copy and paste
+
```

Comment:
TinyMCE ships with copy and paste functionality that helps clean up content by default. In addition, the TinyMCE PowerPaste plugin adds extra functionality and control. 

^ some kind of intro like that? I can't think of the right words

Reply:
We shouldn't do that, as no other page here does that. If we must have it then it needs to be done on other pages and that's definitely out of scope.

Reply:
I'm happy to rework the PowerPaste section below if needed, but I definitely don't want an intro section as that would make this page very inconsitent.

Reply:
Hm, fair. I couldn't think of another way to make it clearer that there was powerpaste and core paste, using the current structure is all. Oh. Unless the PowerPaste section begins with "TinyMCE has paste functionality built in, but the PowerPaste plugin..." kind of thing? I would like the following headings to be followed by something that clarifies what belongs to powerpaste and what's paste too, but words really were failing me

Reply:
Like I said, if we're doing that then we need to do it elsewhere as well. I'm more than happy to change the PowerPaste section as much as needed and starting with `TinyMCE has paste functionality built in, but the PowerPaste plugin...` sounds like the a good idea there.

Reply:
Another issue I realised with including that type of message below, is some options do also apply to PowerPaste (e.g. `paste_as_text`, `paste_tab_spaces`, `paste_preprocess` and `paste_postprocess`). As such, including that would likely only lead to confusion.

Reply:
I've reword the PowerPaste spiel at the top to start with something like the above for now.

Reply:
Added the admonition to try and clarify this as discussed on slack.